src/main/java/com/exadel/project/internship/dto/InternshipInfoDTO.java

```diff
@@ -0,0 +1,30 @@
+package com.exadel.project.internship.dto;
+
+import com.exadel.project.internship.entity.Format;
+import com.exadel.project.internship.entity.InternshipType;
+import com.exadel.project.internship.entity.Subject;
+import com.exadel.project.skill.entity.Skill;
+import lombok.Data;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public class InternshipInfoDTO {
```

Comment:
Maybe it's better to rename into **InternshipDetailsDTO**

Reply:
Done